ts/store/reducers/entities/services/index.ts

```diff
@@ -154,6 +154,17 @@ const getServices = (
     );
 };
 
+// A selector providing all services organized in sections
+export const allServicesSectionsSelector = createSelector(
```

Comment:
I guess it's recommended to wait the merge of this https://github.com/teamdigitale/io-app/pull/1327 before
So we will have only visible services, is it right?

Reply:
As you prefer. In both case the search results will be compatible with the tab content.

Reply:
At this time this search shows all services sections included those ones have not visible services. I think it's good to wait the PR that fixes the visibility issue and then extends that behavior also in here